packages/client-sdk-nodejs/test/unit/cache-service-error-mapper.test.ts

```diff
@@ -27,13 +27,19 @@ const generateServiceError = (status: Status): ServiceError => {
   };
 };
 
+// we need this so that we can reset the resolved/rejected values between tests, because we are using the `no-undefined`
+// lint rule to give our users better support for null vs undefined
+function returnUndefined(): undefined {
+  return;
+}
+
 describe('CacheServiceErrorMapper', () => {
   const cacheServiceErrorMapper = new CacheServiceErrorMapper(false);
   let resolved: SdkError | unknown;
   let rejected: SdkError | unknown;
   beforeEach(() => {
-    resolved = undefined;
-    rejected = undefined;
+    resolved = returnUndefined();
+    rejected = returnUndefined();
```

Comment:
Hmm, this also feels like acrobatics that make me reconsider my stance.

Reply:
yeah it's janky, but in this case it's just some test setup logic.